src/libcore/alloc.rs

```diff
@@ -64,8 +64,9 @@ impl Layout {
     ///    must not overflow (i.e., the rounded value must be less than
     ///    `usize::MAX`).
     #[stable(feature = "alloc_layout", since = "1.28.0")]
+    #[rustc_const_unstable(feature="const_alloc",issue="0")]
```

Comment:
stylistic nit: add spaces around `=` and after `,`

```suggestion
    #[rustc_const_unstable(feature = "const_alloc", issue = "0")]
```

(also apply to all other `rustc_const_unstable` attributes in this file)